src/app/rpmostree-builtin-status.cxx

```diff
@@ -113,8 +113,7 @@ print_packages (const char *k, guint max_key_len,
       if (omit_pkgs != NULL && g_strv_contains (omit_pkgs, pkg))
         continue;
 
-      auto quoted = rpmostreecxx::maybe_shell_quote (pkg);
```

Comment:
There are many other callers of `print_packages` which still rely on this. To answer your comment, the use case for this is that "packages" in some context is a loose term, and actually supports the same syntax dnf does, such as arbitrary provides or reldeps (e.g. this is valid: `rpm-ostree install "foo > 2.1"`).

Maybe what we could do instead is add a new `quote` gboolean parameter to `print_packages` so that way the callers can say if quotes may be necessary. (And then we could pass `FALSE` for the grouped packages).

Reply:
Agreed on the additional function parameter. I'll update the PR shortly.